zebra-network/src/peer_set/initialize/tests/vectors.rs

```diff
@@ -26,10 +34,24 @@ use Network::*;
 async fn local_listener_unspecified_port_unspecified_addr() {
     zebra_test::init();
 
+    if env::var_os("ZEBRA_SKIP_NETWORK_TESTS").is_some() {
```

Comment:
Minor: Should the environment variable names be in constants to make them easier to change and/or avoid some tests not being skipped due to possible different names?

Reply:
Yeah we can do that by adding constants to zebra_test

Reply:
This is fixed in 3bc941cf958c910c9b037cab07f7aa4aef5fbc15..c987cd48c6797c4d13ad7c485b9f76390e74e56d (5 commits).

I split the search and replace commits from the manual cleanup commits, but they should be easy enough to review together.